### PR TITLE
Consistently use openssl.cnf

### PR DESCRIPTION
**Pull request by teor2345**

### CODE REVIEW

File: CAInstructions.markdown [hunk 1]
```diff
@@ -93,7 +93,7 @@ Add the root certificate and key to the destined folders:
 Create a \<user\> key and a certificate signing request in one step: (Enter \<usr_common_name\> when prompted and use a random passphrase)
 
 ```
-    openssl req -newkey rsa:4096 -keyout <usr_common_name>.key -out <usr_common_name>.csr -config openssl.cnf -days 3650
+    openssl req -newkey rsa:4096 -keyout <usr_common_name>.key -out <usr_common_name>.csr -config <openssl_configuration_file> -days 3650
```
Comment (AkshayaMani) [inline CAInstructions.markdown]: May be use PSC/openssl.cnf ?

As we have used openssl.cnf in Configuration file section:

'''
Create a CA configuration file (refer PSC/openssl.cnf). This is optional, but is very convenient while issuing multiple certificates. 
 
 Edit openssl.cnf, and change the following fields: (Here, directory \<usr\> must be directory TS, CP, or DP depending on the keys you want to generate.)
'''

File: CAInstructions.markdown [hunk 2]
```diff
@@ -106,7 +106,7 @@ Create the insecure key, the one without a passphrase, and interchange the key n
 Using the CSR, generate a certificate signed by the CA:
 
 ```
-    openssl ca -in <usr_common_name>.csr -config openssl.cnf
+    openssl ca -in <usr_common_name>.csr -config <openssl_configuration_file>
```
Comment (AkshayaMani) [inline CAInstructions.markdown]: May be use PSC/openssl.cnf ?

As we have used openssl.cnf in Configuration file section:

'''
Create a CA configuration file (refer PSC/openssl.cnf). This is optional, but is very convenient while issuing multiple certificates.

Edit openssl.cnf, and change the following fields: (Here, directory <usr> must be directory TS, CP, or DP depending on the keys you want to generate.)
'''

**Comment by teor2345** [comment]
I update the branch to use openssl.cnf, and rebased it on the latest master

**Comment by AkshayaMani** [comment]
I do not see any requests here. Can we close it?

**Pull request merged**